### DISCUSSION TITLE
FYI 3DConnexion just implemented an Unreal Plugin for 3DXWare

### DISCUSSION THREAD

**Discussion by brybalicious** [microdee/OpenUnrealSpaceMouse#25] [General]
Apparently new drivers 10.7.1 include a 3DXUnreal plugin... wonder how different they feel

https://17osrg460vnh32txvs1x75a9-wpengine.netdna-ssl.com/uk/wp-content/uploads/sites/1/2020/12/Release_Notes-Windows-20201215-1.pdf

**Comment by microdee** [comment]
oh great timing. Well this plugin is still useful for the very niche users who doesn't want 3DxWare, if you try it out tell me how different it is

**Comment by microdee** [comment]
Here's a comparison chart between the two plugins https://github.com/microdee/UE4-SpaceMouse/blob/master/DISCLAIMER.md

**Comment by brybalicious** [comment]
Thanks! TBH, I couldn't get the 3dxware working. It won't build because of some missing libs or something, so I'm just going to continue using your excellent plugin. If it ain't broke, don't fix it. Just wish the Orbit No Roll function worked without jittering.

Thanks for wrilting that up

**Reply by microdee** [reply]
yeah I found it weird that they didn't have at least the UE4 specific source in their plugins (only the Build.cs), I would have been curious about implementation differences.
